cli/src/base/ContentDirectoryCommandBase.ts

```diff
@@ -163,4 +244,33 @@ export default abstract class ContentDirectoryCommandBase extends AccountsComman
       return columns
     }, {} as Record<string, { value: Codec; type: string }>)
   }
+
+  async parseToKnownEntityJson<T>(entity: Entity, entityClass?: Class): Promise<FlattenRelations<T>> {
```

Comment:
That's true, I actually forgot to remove this, the purpose of the optional argument was to avoid an additional api call, but after I implmeneted `_cdClassesCache` it is no longer necessary.